pom.xml

```diff
@@ -8,7 +8,7 @@
 
     <groupId>ConfigHub</groupId>
     <artifactId>ConfigHub</artifactId>
-    <version>1.8.4</version>
+    <version>1.9.0</version>
```

Comment:
there's a few other files that you need to edit, if you just search for 1.8.4 you'll find it.